e2e/s3/swupdate/Dockerfile

```diff
@@ -52,20 +52,20 @@ RUN mkdir -p /app && apt update && apt install -y build-essential \
 
 WORKDIR /app
 
-RUN git clone https://github.com/sbabic/swupdate --depth 1
+RUN git clone https://github.com/sbabic/swupdate && cd swupdate && git reset --hard cad85ea
```

Comment:
Probably best to avoid short hashes here for security as they aren't collision resistant AFAIU.

Reply:
Updated to full commit hash.